pkg/storaged/stratis/create-dialog.jsx

```diff
@@ -28,6 +28,23 @@ import { validate_url, get_tang_adv } from "../crypto/tang.jsx";
 
 const _ = cockpit.gettext;
 
+function manager_create_pool(name, devs, key_desc, clevis_info) {
+    if (client.stratis_interface_revision < 8) {
+        return client.stratis_manager.CreatePool(name,
+                                                 devs,
+                                                 key_desc ? [true, key_desc] : [false, ""],
+                                                 clevis_info ? [true, clevis_info] : [false, ["", ""]]);
```

Comment:
These 5 added lines are not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-21927-7efac793-20250814-135251-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)